### PR TITLE
Allow resolv.conf to be managed for Ubuntu AIO

### PR DESCRIPTION
**Pull request by assumptionsandg**
Fix Ubuntu AIOs in CI by allowing resolv.conf to be managed by Kayobe.

### CODE REVIEW

**Comment by assumptionsandg** [comment]
Solved in #854 for CI

Turns out DNS config wasn't being applied as networkd config was being set for ens3 rather than enp3s0. Still broken for manual deployment AFAIK.

**Comment by markgoddard** [comment]
Should this PR be closed?

**Comment by markgoddard** [comment]
> Solved in #854 for CI
> 
> Turns out DNS config wasn't being applied as networkd config was being set for ens3 rather than enp3s0. Still broken for manual deployment AFAIK.

Why is it broken in a manual deployment?

**Comment by assumptionsandg** [comment]
> > Solved in #854 for CI
> > Turns out DNS config wasn't being applied as networkd config was being set for ens3 rather than enp3s0. Still broken for manual deployment AFAIK.
> 
> Why is it broken in a manual deployment?

networkd configuration for enp3s0 isn't being templated in when deploying manually via the automated script, hence the need for resolv.conf to be managed by Kayobe in these cases as DNS breaks when the device is left unmanaged.

**Comment by markgoddard** [comment]
I have now hit this in an Ubuntu VM and can see the issue you're trying to address.

File: etc/kayobe/environments/ci-aio/stackhpc-ci.yml [hunk 1]
```diff
@@ -8,9 +8,10 @@ kolla_docker_namespace: stackhpc-dev
 ###############################################################################
 # Network configuration.
 
-# Don't touch resolv.conf: use Neutron DNS for accessing Pulp server via
-# hostname.
```
Comment (markgoddard) [inline etc/kayobe/environments/ci-aio/stackhpc-ci.yml:12]: I think we don't actually make use of this - we write out an entry in /etc/hosts. I suggest we just remove all of this config.

Comment (holliefae) [reply etc/kayobe/environments/ci-aio/stackhpc-ci.yml:12]: To clarify do you mean remove the resolv config entirely or just remove the hostname resolv_domain?

Comment (markgoddard) [reply etc/kayobe/environments/ci-aio/stackhpc-ci.yml:12]: We can remove everything in this `Network configuration` section

Comment (holliefae) [reply etc/kayobe/environments/ci-aio/stackhpc-ci.yml:12]: I'll close this one and make a pr.

**Comment by assumptionsandg** [comment]
Continued in #920

**Pull request closed without merging**